extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -430,14 +431,14 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		// the top we'll expand nodes until we find the parent of the Notebook we're looking for
 		// get the children of root node and expand the nodes to the notebook level.
 		await this.getChildren(parentBook.rootNode);
-		// The path to the parent of the Notebook we're looking for (this is the node we're looking to expand)
-		const parentPath = notebookPath.substring(0, notebookPath.lastIndexOf(path.posix.sep));
+		// The path to the Notebook we're looking for (these are the nodes we're looking to expand)
+		const notebookFolders = notebookPath.split(path.posix.sep);
```

Comment:
We don't seem to be verifying that the path is actually separated by / - so anyone could pass in a windows-style path and this would just break. Would prefer we at least do the whole "replace \ with /" thing at the beginning of this function in that case

(insert same comment here about us having to do that logic EVERYWHERE and making it so that stuff like this is super easy to screw up)